nixos/modules/services/x11/desktop-managers/gnome3.nix

```diff
@@ -177,6 +178,8 @@ in
                 description = "The executable of the window manager to use.";
                 example = "\${pkgs.haskellPackages.xmonad}/bin/xmonad";
               };
+
+              enableGnomePanel = mkEnableOption "the GNOME panel in this session";
```

Comment:
Maybe it should be opt-out since panel is part of the flashback experience.

Reply:
Done.